<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnBrowser">
           <property name="text">
            <string>浏览文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearLst">
           <property name="text">
            <string>清空列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCpsSlt">
           <property name="text">
            <string>压缩所选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDecompress">
           <property name="text">
            <string>解压所选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRecover">
           <property name="text">
            <string>恢复备份</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteBackup">
           <property name="text">
            <string>删除备份文件</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>多线程压缩</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spbMultiThread">
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>压缩级别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbbSpeed">
             <property name="currentText">
              <string>default</string>
             </property>
             <item>
              <property name="text">
               <string>default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1(速度最快)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bset(效率最高)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>备份文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbbBackup">
             <item>
              <property name="text">
               <string>保留备份</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>不保留备份文件</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTreeView" name="lstFileList">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QProgressBar" name="pgsGlobal">
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%p%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtLog">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest">
   <property name="text">
    <string>测试</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除(非源文件)</string>
   </property>
   <property name="toolTip">
    <string>删除文件,保留源文件</string>
   </property>
  </action>
  <action name="actionDeleteFile">
   <property name="text">
    <string>删除(源文件)</string>
   </property>
   <property name="toolTip">
    <string>删除文件,并且删除源文件</string>
   </property>
  </action>
  <action name="actionClearTxt">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空错误信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
